proposals/lip-0041.md

```diff
@@ -250,43 +555,101 @@ The Auth module does not execute any logic during the genesis state finalization
 
 #### Transaction Verification
 
-As part of the verification of a transaction `trs`, the following checks are applied. If the checks fail the transaction is invalid and the whole block is discarded.
+As part of the verification of a transaction `trs`, we do the following:
 
-* Check that the transaction nonce is equal to the account nonce by returning `isValidNonce(trs)`.
+* Check that the transaction nonce is equal to the account nonce or to 0 if the account does not exist. If the checks fail the transaction is invalid and the whole block is discarded.
```

Comment:
I think we could also remove this point as this is already specified in the code below. Maybe we want to still keep the notice about the transaction pool below.

Reply:
removed